core/modules/system/system.menu.inc

```diff
@@ -146,6 +147,17 @@ function system_menu_block_form($config) {
       '!menu_settings' => $link,
     )),
   );
+  $form['toggle_text'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Toggle button text'),
+    '#default_value' => $config['toggle_text'],
+    '#description' => t('Text to display beside the toggle button. Default: <em>Menu</em>.'),
```

Comment:
Can you clarify? The toggle text default is already in `system_menu_block_defaults()`:

```
'toggle_text' => 'Menu',
```

Reply:
@laryn, I think @klonos means something like this:

```suggestion
    '#description' => t('Text to display beside the toggle button. Default: %toggle_text.', array('%toggle_text' => system_menu_block_defaults($config['menu_name'])['toggle_text'])),
```

I've squeezed it into a single line of code for suggestion purposes, but it might be more readable to create a `$toggle_text` variable just before the form entry, then plug it into the `#description` `t()` call. And note that since we're using `t()`, we can use a `%` variable to get the emphasis, so we don't need to add HTML inside the translatable text.